content/registration-system/_index.md

```diff
@@ -10,4 +10,9 @@ weight = 45
 
 # Registration system
 
-Information on the registration system used by [anmalan-forsranningen.utn.se](https://anmalan-forsranningen.utn.se)
+Information on the registration system used by [anmalan-forsranningen.utn.se](https://anmalan-forsranningen.utn.se).
```

Comment:
You should mention that this documentation goes through both how the system works from both a user and technical perspective since we mention various parts from the code